docs/source/release/v6.2.0/framework.rst

```diff
@@ -86,15 +81,13 @@ Bugfixes
 SliceViewer
 -----------
 
-Improvements
-############
-
 Bugfixes
 ########
-- Fix cursor tracking from getting stuck and displaying incorrect signals when viewing MDHistogram workspaces in :ref:`sliceviewer`.
-- Added parser for input Names to :ref:`algm-CreateMDHistoWorkspace` to allow inputs such as `Names='[H,0,0],[0,K,0],[0,0,L]'`.
-- Fixed bug in :ref:`algm-ConvertToMDMinMaxLocal` where wrong min max calculated if the workspace includes monitor spectra or spectra without any detectors
+- Fixed cursor tracking from getting stuck and displaying incorrect signals when viewing MDHistogram workspaces in :ref:`sliceviewer`.
 - Fix bug in :ref:`CalculateMultipleScattering <algm-CalculateMultipleScattering>` where detector position was incorrectly determined on a workspace where the workspace index didn't match the detector
-  index eg if the workspace was loaded with SpectrumMin specified to exclude some monitors
+  index e.g. if the workspace was loaded with ``SpectrumMin`` specified to exclude some monitors.
+- Fixed bug in :ref:`algm-ConvertToMDMinMaxLocal` where wrong min max calculated if the workspace includes monitor spectra or spectra without any detectors.
+- Added parser for input Names to :ref:`algm-CreateMDHistoWorkspace` to allow inputs such as `Names='[H,0,0],[0,K,0],[0,0,L]'`.
```

Comment:
Can I ask for these 3 bullet points on CalculateMultipleScattering, ConvertToMDMinMaxLocal, CreateMDHistoWorkspace to be moved up to the Algorithms/Bugfixes section? One of them is mine, but they aren't really sliceviewer related.